### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by metsma**
WE2-501

Signed-off-by: Raul Metsma <raul@metsma.ee>

### CODE REVIEW

File: src/ui/webeiddialog.cpp [hunk 1]
```diff
@@ -99,6 +101,10 @@ WebEidDialog::WebEidDialog(QWidget* parent) : WebEidUI(parent), ui(new Private)
                                        ui->pinEntryTimeoutProgressBar->maximum());
     connect(ui->pinTimeoutTimer, &QTimeLine::frameChanged, ui->pinEntryTimeoutProgressBar,
             &QProgressBar::setValue);
+    connect(ui->pinTimeoutTimer, &QTimeLine::frameChanged, ui->pinTimeRemaining, [this](int value) {
+        ui->pinTimeRemaining->setText(
+            tr("Time remaining: <b>%1</b>").arg(ui->pinEntryTimeoutProgressBar->maximum() - value));
```
Comment (mrts) [inline src/ui/webeiddialog.cpp:106]: Should we convert this value to minutes & seconds?

File: src/ui/certificatewidget.cpp [hunk 2]
```diff
@@ -31,35 +32,85 @@
...
     const auto certInfo = cardCertPinInfo.certInfo;
-    info->setText(
-        CertificateWidget::tr("<b>%1</b><br />Issuer: %2<br />Valid: %3 to %4")
-            .arg(certInfo.subject, certInfo.issuer, certInfo.effectiveDate, certInfo.expiryDate));
+    QString warning;
+    auto showRed = [](const QString& text, bool show) {
+        return show ? QStringLiteral("<span style=\"color: #CD2541\">%1</span>").arg(text) : text;
+    };
+    if (certInfo.notEffective) {
+        warning = showRed(CertificateWidget::tr(" (Not effective)"), true);
```
Comment (mrts) [inline src/ui/certificatewidget.cpp:83]: I think this can be simplified, I'll submit a follow-up patch.

**Review by mrts — CHANGES_REQUESTED** [review]

File: src/controller/command-handlers/certificatereader.cpp [hunk 3]
```diff
@@ -52,19 +45,9 @@ getCertificateWithStatusAndInfo(const CardInfo::ptr& card, const CertificateType
               "Invalid certificate returned by electronic ID " + card->eid().name());
     }
 
-    auto certificateStatus = CertificateStatus::VALID;
-
-    if (certificate.isNull()) {
```
[the remark this answers is not in the document]
Comment (metsma) [reply src/controller/command-handlers/certificatereader.cpp:57]: Define invalid certificate?

File: src/ui/webeiddialog.cpp (hunk 1 above)
Comment (metsma) [reply src/ui/webeiddialog.cpp:106]: It is seconds

File: src/controller/command-handlers/certificatereader.cpp (hunk 3 above)
[the remark this answers is not in the document]
Comment (mrts) [reply src/controller/command-handlers/certificatereader.cpp:57]: I'd define certificate as invalid when `certificate.isNull()` is true. But thanks, code review view did not show the `isNull()` check above, so this was a duplicate and all is good.

**Review by mrts — APPROVED** [review]

**Pull request merged**